README.md

```diff
@@ -1,39 +1,39 @@
-# Ubuntu 22.04 based MLflow image v2.1.1
-This repository regularly builds new [Mlflow](https://github.com/mlflow/mlflow) image base on LTS Ubuntu 22.04. The process weekly downloads the upstream MLflow v2.1.1 and changes the base for Docker image with the Dockerfile you can see in this repository. After the new image is built the ci runs simple tests against the image and runs the [trivy](https://github.com/aquasecurity/trivy) vulnerability scan. The results are published to the security tab in the repository (only visible to admins). 
 
-The image is currently published [here](https://hub.docker.com/r/charmed/base-mlflow)
-You can use this image seamlesly as any other python Docker image by specifying it as a base with.
+# Ubuntu 22.04 based MLflow image v2.15.0
+This repository regularly builds a new [MLflow](https://github.com/mlflow/mlflow) image based on LTS Ubuntu 22.04. After the new image is built, the CI runs simple tests against the image and runs the [trivy](https://github.com/aquasecurity/trivy) vulnerability scan. 
 
-```
-FROM charmed/base-mlflow:latest
+The image is currently published [here](https://hub.docker.com/r/charmedkubeflow/mlflow)
+You can use this image seamlessly as any other Python Docker image by specifying it as a base with:
+
+```dockerfile
+FROM charmedkubeflow/mlflow:latest
 ```
 
 # MLflow ROCK OCI image
 
-The following tools are required to build ROCK image manually:
-- `rockcraft' - A tool to create OCI images.
+The following tools are required to build the ROCK image manually:
+- `rockcraft` - A tool to create OCI images.
 - `skopeo` - A tool to operate on container images and registries.
 
 To install tools:
-```
+```bash
 sudo snap install rockcraft --classic --edge
 sudo snap install skopeo --edge --devmode
 ```
 
-To build ROCK image manually:
-```
+To build the ROCK image manually:
+```bash
 rockcraft pack
 ```
 
-To copy resulting image `base-mflow_v2.1.1_amd64.rock` to Docker:
+To copy the resulting image `base-mlflow_v2.15.0_amd64.rock` to Docker:
+```bash
+sudo skopeo --insecure-policy copy oci-archive:base-mlflow_v2.15.0_amd64.rock docker-daemon:base-mlflow_v2.15.0_amd64.rock:rock
 ```
```

Comment:
Should we replace those commands with the new tox environments? This way, we won't need to maintain those too.